===================================================================
--- lib/aave-v3-origin/src/contracts/interfaces/IDefaultInterestRateStrategyV2.sol
+++ lib/aave-v3-origin-private/src/contracts/interfaces/IDefaultInterestRateStrategyV2.sol
@@ -88,9 +88,9 @@
      */
     function MAX_OPTIMAL_POINT() external view returns (uint256);
 
     /**
-     * notice Returns the full InterestRateData object for the given reserve, in ray
+     * @notice Returns the full InterestRateData object for the given reserve, in ray
      *
      * @param reserve The reserve to get the data of
      *
      * @return The InterestRateDataRay object for the given reserve
@@ -99,9 +99,9 @@
         address reserve
     ) external view returns (InterestRateDataRay memory);
 
     /**
-     * notice Returns the full InterestRateDataRay object for the given reserve, in bps
+     * @notice Returns the full InterestRateData object for the given reserve, in bps
      *
      * @param reserve The reserve to get the data of
      *
      * @return The InterestRateData object for the given reserve
