Index: settings
===================================================================
--- settings
+++ settings
@@ -1,21 +1,22 @@
 {
   "remappings": [
     "aave-address-book/=lib/aave-helpers/lib/aave-address-book/src/",
     "aave-helpers/=lib/aave-helpers/",
-    "aave-v3-origin-tests/=lib/aave-v3-origin/tests/",
-    "aave-v3-origin/=lib/aave-v3-origin/src/",
+    "aave-v3-origin-tests/=lib/aave-v3-origin-private/tests/",
+    "aave-v3-origin/=lib/aave-v3-origin-private/src/",
     "erc4626-tests/=lib/aave-helpers/lib/solidity-utils/lib/openzeppelin-contracts-upgradeable/lib/erc4626-tests/",
-    "forge-std/=lib/forge-std/src/",
+    "forge-std/=lib/aave-helpers/lib/forge-std/src/",
     "openzeppelin-contracts-upgradeable/=lib/aave-helpers/lib/aave-address-book/lib/aave-v3-origin/lib/solidity-utils/lib/openzeppelin-contracts-upgradeable/",
     "openzeppelin-contracts/=lib/aave-helpers/lib/aave-address-book/lib/aave-v3-origin/lib/solidity-utils/lib/openzeppelin-contracts-upgradeable/lib/openzeppelin-contracts/",
     "solidity-utils/=lib/aave-helpers/lib/aave-address-book/lib/aave-v3-origin/lib/solidity-utils/src/",
     "lib/aave-helpers/:aave-address-book/=lib/aave-helpers/lib/aave-address-book/src/",
     "lib/aave-helpers/:solidity-utils/=lib/aave-helpers/lib/aave-address-book/lib/aave-v3-origin/lib/solidity-utils/src/",
-    "@openzeppelin/contracts-upgradeable/=lib/aave-v3-origin/lib/solidity-utils/lib/openzeppelin-contracts-upgradeable/contracts/",
-    "@openzeppelin/contracts/=lib/aave-v3-origin/lib/solidity-utils/lib/openzeppelin-contracts-upgradeable/lib/openzeppelin-contracts/contracts/",
-    "ds-test/=lib/aave-v3-origin/lib/forge-std/lib/ds-test/src/",
-    "halmos-cheatcodes/=lib/aave-v3-origin/lib/solidity-utils/lib/openzeppelin-contracts-upgradeable/lib/halmos-cheatcodes/src/"
+    "@openzeppelin/contracts-upgradeable/=lib/aave-v3-origin-private/lib/solidity-utils/lib/openzeppelin-contracts-upgradeable/contracts/",
+    "@openzeppelin/contracts/=lib/aave-v3-origin-private/lib/solidity-utils/lib/openzeppelin-contracts-upgradeable/lib/openzeppelin-contracts/contracts/",
+    "aave-v3-origin-private/=lib/aave-v3-origin-private/",
+    "ds-test/=lib/aave-v3-origin-private/lib/forge-std/lib/ds-test/src/",
+    "halmos-cheatcodes/=lib/aave-v3-origin-private/lib/solidity-utils/lib/openzeppelin-contracts-upgradeable/lib/halmos-cheatcodes/src/"
   ],
   "optimizer": {
     "enabled": true,
     "runs": 200
@@ -37,11 +38,6 @@
       ]
     }
   },
   "evmVersion": "cancun",
-  "viaIR": false,
-  "libraries": {
-    "lib/aave-v3-origin/src/contracts/instances/PoolConfiguratorInstance.sol": {
-      "ConfiguratorLogic": "0x6e2afd57a161d12f34f416c29619bfeacac8aa18"
-    }
-  }
+  "viaIR": false
 }
\ No newline at end of file
