===================================================================
--- src/contracts/protocol/libraries/logic/ConfiguratorLogic.sol
+++ lib/aave-v3-origin-private/src/contracts/protocol/libraries/logic/ConfiguratorLogic.sol
@@ -29,9 +29,9 @@
      */
     function executeInitReserve(
         IPool pool,
         ConfiguratorInputTypes.InitReserveInput calldata input
-    ) external {
+    ) internal {
         // It is an assumption that the asset listed is non-malicious, and the external call doesn't create re-entrancies
         uint8 underlyingAssetDecimals = IERC20Metadata(input.underlyingAsset)
             .decimals();
         require(underlyingAssetDecimals > 5, Errors.InvalidDecimals());
@@ -105,9 +105,9 @@
      */
     function executeUpdateAToken(
         IPool cachedPool,
         ConfiguratorInputTypes.UpdateATokenInput calldata input
-    ) external {
+    ) internal {
         address aTokenAddress = cachedPool.getReserveAToken(input.asset);
 
         uint256 decimals = cachedPool
             .getConfiguration(input.asset)
@@ -144,9 +144,9 @@
      */
     function executeUpdateVariableDebtToken(
         IPool cachedPool,
         ConfiguratorInputTypes.UpdateDebtTokenInput calldata input
-    ) external {
+    ) internal {
         address variableDebtTokenAddress = cachedPool
             .getReserveVariableDebtToken(input.asset);
 
         uint256 decimals = cachedPool
