===================================================================
--- src/contracts/protocol/pool/PoolConfigurator.sol
+++ lib/aave-v3-origin-private/src/contracts/protocol/pool/PoolConfigurator.sol
@@ -96,14 +96,8 @@
         }
     }
 
     /// @inheritdoc IPoolConfigurator
-    function dropReserve(address asset) external override onlyPoolAdmin {
-        _pool.dropReserve(asset);
-        emit ReserveDropped(asset);
-    }
-
-    /// @inheritdoc IPoolConfigurator
     function updateAToken(
         ConfiguratorInputTypes.UpdateATokenInput calldata input
     ) external override onlyPoolAdmin {
         ConfiguratorLogic.executeUpdateAToken(_pool, input);
@@ -292,20 +286,8 @@
         _pool.setConfiguration(asset, currentConfig);
     }
 
     /// @inheritdoc IPoolConfigurator
-    function setBorrowableInIsolation(
-        address asset,
-        bool borrowable
-    ) external override onlyRiskOrPoolAdmins {
-        DataTypes.ReserveConfigurationMap memory currentConfig = _pool
-            .getConfiguration(asset);
-        currentConfig.setBorrowableInIsolation(borrowable);
-        _pool.setConfiguration(asset, currentConfig);
-        emit BorrowableInIsolationChanged(asset, borrowable);
-    }
-
-    /// @inheritdoc IPoolConfigurator
     function setReservePause(
         address asset,
         bool paused,
         uint40 gracePeriod
@@ -368,50 +350,8 @@
         _pool.syncRatesState(asset);
     }
 
     /// @inheritdoc IPoolConfigurator
-    function setDebtCeiling(
-        address asset,
-        uint256 newDebtCeiling
-    ) external override onlyRiskOrPoolAdmins {
-        DataTypes.ReserveConfigurationMap memory currentConfig = _pool
-            .getConfiguration(asset);
-
-        uint256 oldDebtCeiling = currentConfig.getDebtCeiling();
-        if (_checkAssetIsCollateral(asset) && oldDebtCeiling == 0) {
-            _checkNoSuppliers(asset);
-        }
-        currentConfig.setDebtCeiling(newDebtCeiling);
-        _pool.setConfiguration(asset, currentConfig);
-
-        if (newDebtCeiling == 0) {
-            _pool.resetIsolationModeTotalDebt(asset);
-        }
-
-        emit DebtCeilingChanged(asset, oldDebtCeiling, newDebtCeiling);
-    }
-
-    /// @inheritdoc IPoolConfigurator
-    function setSiloedBorrowing(
-        address asset,
-        bool newSiloed
-    ) external override onlyRiskOrPoolAdmins {
-        if (newSiloed) {
-            _checkNoBorrowers(asset);
-        }
-        DataTypes.ReserveConfigurationMap memory currentConfig = _pool
-            .getConfiguration(asset);
-
-        bool oldSiloed = currentConfig.getSiloedBorrowing();
-
-        currentConfig.setSiloedBorrowing(newSiloed);
-
-        _pool.setConfiguration(asset, currentConfig);
-
-        emit SiloedBorrowingChanged(asset, oldSiloed, newSiloed);
-    }
-
-    /// @inheritdoc IPoolConfigurator
     function setBorrowCap(
         address asset,
         uint256 newBorrowCap
     ) external override onlyRiskOrPoolAdmins {
@@ -458,9 +398,10 @@
         uint8 categoryId,
         uint16 ltv,
         uint16 liquidationThreshold,
         uint16 liquidationBonus,
-        string calldata label
+        string calldata label,
+        bool isolated
     ) external override onlyRiskOrPoolAdmins {
         require(ltv != 0, Errors.InvalidEmodeCategoryParams());
         require(liquidationThreshold != 0, Errors.InvalidEmodeCategoryParams());
 
@@ -487,8 +428,9 @@
         DataTypes.EModeCategoryBaseConfiguration memory categoryData;
         categoryData.ltv = ltv;
         categoryData.liquidationThreshold = liquidationThreshold;
         categoryData.liquidationBonus = liquidationBonus;
+        categoryData.isolated = isolated;
         categoryData.label = label;
 
         _pool.configureEModeCategory(categoryId, categoryData);
         emit EModeCategoryAdded(
@@ -498,8 +440,9 @@
             liquidationBonus,
             address(0),
             label
         );
+        emit EModeCategoryIsolationChanged(categoryId, isolated);
     }
 
     /// @inheritdoc IPoolConfigurator
     function setAssetCollateralInEMode(
@@ -630,8 +573,17 @@
         );
     }
 
     /// @inheritdoc IPoolConfigurator
+    function setEModeCategoryIsolated(
+        uint8 categoryId,
+        bool isolated
+    ) external onlyRiskOrPoolOrEmergencyAdmins {
+        _pool.configureEModeCategoryIsolated(categoryId, isolated);
+        emit EModeCategoryIsolationChanged(categoryId, isolated);
+    }
+
+    /// @inheritdoc IPoolConfigurator
     function setReserveInterestRateData(
         address asset,
         bytes calldata rateData
     ) external onlyRiskOrPoolAdmins {
@@ -695,13 +647,8 @@
     ) external view override returns (uint256) {
         return _pendingLtv[asset];
     }
 
-    /// @inheritdoc IPoolConfigurator
-    function getConfiguratorLogic() external pure returns (address) {
-        return address(ConfiguratorLogic);
-    }
-
     function _setReserveLtvzero(
         address asset,
         bool ltvZero,
         DataTypes.ReserveConfigurationMap memory currentConfig
