===================================================================
--- src/contracts/instances/PoolConfiguratorInstance.sol
+++ lib/aave-v3-origin-private/src/contracts/instances/PoolConfiguratorInstance.sol
@@ -8,9 +8,9 @@
  * @author BGD Labs
  * @notice Instance of the PoolConfigurator of the Aave protocol
  */
 contract PoolConfiguratorInstance is PoolConfigurator {
-    uint256 public constant CONFIGURATOR_REVISION = 7;
+    uint256 public constant CONFIGURATOR_REVISION = 8;
 
     /// @inheritdoc VersionedInitializable
     function getRevision() internal pure virtual override returns (uint256) {
         return CONFIGURATOR_REVISION;
