===================================================================
--- lib/aave-v3-origin/src/contracts/protocol/libraries/configuration/EModeConfiguration.sol
+++ lib/aave-v3-origin-private/src/contracts/protocol/libraries/configuration/EModeConfiguration.sol
@@ -27,8 +27,9 @@
             require(
                 reserveIndex < ReserveConfiguration.MAX_RESERVES_COUNT,
                 Errors.InvalidReserveIndex()
             );
+            // forge-lint: disable-next-line(incorrect-shift, unsafe-typecast)
             uint128 bit = uint128(1 << reserveIndex);
             if (enabled) {
                 return bitmap | bit;
             } else {
