===================================================================
--- lib/aave-v3-origin/src/contracts/protocol/libraries/helpers/Errors.sol
+++ lib/aave-v3-origin-private/src/contracts/protocol/libraries/helpers/Errors.sol
@@ -45,40 +45,33 @@
     error SpecifiedCurrencyNotBorrowedByUser(); // 'User did not borrow the specified currency'
     error InconsistentFlashloanParams(); // 'Inconsistent flashloan parameters'
     error BorrowCapExceeded(); // 'Borrow cap is exceeded'
     error SupplyCapExceeded(); // 'Supply cap is exceeded'
-    error DebtCeilingExceeded(); // 'Debt ceiling is exceeded'
-    error UnderlyingClaimableRightsNotZero(); // 'Claimable rights over underlying not zero (aToken supply or accruedToTreasury)'
-    error VariableDebtSupplyNotZero(); // 'Variable debt supply is not zero'
     error LtvValidationFailed(); // 'Ltv validation failed'
     error InconsistentEModeCategory(); // 'Inconsistent eMode category'
-    error PriceOracleSentinelCheckFailed(); // 'Price oracle sentinel validation failed'
-    error AssetNotBorrowableInIsolation(); // 'Asset is not borrowable in isolation mode'
     error ReserveAlreadyInitialized(); // 'Reserve has already been initialized'
-    error UserInIsolationModeOrLtvZero(); // 'User is in isolation mode or ltv is zero'
+    error UserHasAssetWithZeroLtv(); // 'User has asset with ltv being zero'
     error InvalidLtv(); // 'Invalid ltv parameter for the reserve'
     error InvalidLiquidationThreshold(); // 'Invalid liquidity threshold parameter for the reserve'
     error InvalidLiquidationBonus(); // 'Invalid liquidity bonus parameter for the reserve'
     error InvalidDecimals(); // 'Invalid decimals parameter of the underlying asset of the reserve'
     error InvalidReserveFactor(); // 'Invalid reserve factor parameter for the reserve'
     error InvalidBorrowCap(); // 'Invalid borrow cap for the reserve'
     error InvalidSupplyCap(); // 'Invalid supply cap for the reserve'
     error InvalidLiquidationProtocolFee(); // 'Invalid liquidation protocol fee for the reserve'
-    error InvalidDebtCeiling(); // 'Invalid debt ceiling for the reserve'
     error InvalidReserveIndex(); // 'Invalid reserve index'
     error AclAdminCannotBeZero(); // 'ACL admin cannot be set to the zero address'
     error InconsistentParamsLength(); // 'Array parameters that should be equal length are not'
     error ZeroAddressNotValid(); // 'Zero address not valid'
     error InvalidExpiration(); // 'Invalid expiration'
     error InvalidSignature(); // 'Invalid signature'
     error OperationNotSupported(); // 'Operation not supported'
-    error DebtCeilingNotZero(); // 'Debt ceiling is not zero'
     error AssetNotListed(); // 'Asset is not listed'
     error InvalidOptimalUsageRatio(); // 'Invalid optimal usage ratio'
     error UnderlyingCannotBeRescued(); // 'The underlying asset cannot be rescued'
     error AddressesProviderAlreadyAdded(); // 'Reserve has already been added to reserve list'
     error PoolAddressesDoNotMatch(); // 'The token implementation pool address and the pool address provided by the initializing pool do not match'
-    error SiloedBorrowingViolation(); // 'User is trying to borrow multiple assets including a siloed one'
+
     error ReserveDebtNotZero(); // the total debt of the reserve needs to be 0
     error FlashloanDisabled(); // FlashLoaning for this asset is disabled
     error InvalidMaxRate(); // The expect maximum borrow rate is invalid
     error WithdrawToAToken(); // Withdrawing to the aToken is not allowed
