===================================================================
--- lib/aave-v3-origin/src/contracts/interfaces/IPool.sol
+++ lib/aave-v3-origin-private/src/contracts/interfaces/IPool.sol
@@ -77,18 +77,8 @@
         bool useATokens
     );
 
     /**
-     * @dev Emitted on borrow(), repay() and liquidationCall() when using isolated assets
-     * @param asset The address of the underlying asset of the reserve
-     * @param totalDebt The total isolation mode debt for the reserve
-     */
-    event IsolationModeTotalDebtUpdated(
-        address indexed asset,
-        uint256 totalDebt
-    );
-
-    /**
      * @dev Emitted when the user selects a certain asset category for eMode
      * @param user The address of the user
      * @param categoryId The category id
      */
@@ -485,16 +475,8 @@
         address variableDebtAddress
     ) external;
 
     /**
-     * @notice Drop a reserve
-     * @dev Only callable by the PoolConfigurator contract
-     * @dev Does not reset eMode flags, which must be considered when reusing the same reserve id for a different reserve.
-     * @param asset The address of the underlying asset of the reserve
-     */
-    function dropReserve(address asset) external;
-
-    /**
      * @notice Accumulates interest to all indexes of the reserve
      * @dev Only callable by the PoolConfigurator contract
      * @dev To be used when required by the configurator, for example when updating interest rates strategy data
      * @param asset The address of the underlying asset of the reserve
@@ -684,8 +666,15 @@
         uint128 ltvzeroBitmap
     ) external;
 
     /**
+     * @notice Sets the isolated flag of an eMode category.
+     * @param id The id of the category
+     * @param isolated True if the eMode should be isolated
+     */
+    function configureEModeCategoryIsolated(uint8 id, bool isolated) external;
+
+    /**
      * @notice Returns the data of an eMode category
      * @dev DEPRECATED use independent getters instead
      * @param id The id of the category
      * @return The configuration data of the category
@@ -741,8 +730,15 @@
         uint8 id
     ) external view returns (uint128);
 
     /**
+     * @notice Returns the isolated flag of an eMode category
+     * @param id The id of the category
+     * @return True if the eMode category is isolated
+     */
+    function getIsEModeCategoryIsolated(uint8 id) external view returns (bool);
+
+    /**
      * @notice Allows a user to use the protocol in eMode
      * @param categoryId The id of the category
      */
     function setUserEMode(uint8 categoryId) external;
@@ -754,15 +750,8 @@
      */
     function getUserEMode(address user) external view returns (uint256);
 
     /**
-     * @notice Resets the isolation mode total debt of the given asset to zero
-     * @dev It requires the given asset has zero debt ceiling
-     * @param asset The address of the underlying asset to reset the isolationModeTotalDebt
-     */
-    function resetIsolationModeTotalDebt(address asset) external;
-
-    /**
      * @notice Sets the liquidation grace period of the given asset
      * @dev To enable a liquidation grace period, a timestamp in the future should be set,
      *      To disable a liquidation grace period, any timestamp in the past works, like 0
      * @param asset The address of the underlying asset to set the liquidationGracePeriod
@@ -884,9 +873,9 @@
         uint8 categoryId,
         address onBehalfOf
     ) external;
 
-    /*
+    /**
      * @notice Returns true if the `positionManager` address is approved to use the position manager role on behalf of the user.
      * @param user The address of the user
      * @param positionManager The address of the position manager
      * @return True if the user is approved to use the position manager, false otherwise
