===================================================================
--- lib/aave-v3-origin/src/contracts/protocol/libraries/math/PercentageMath.sol
+++ lib/aave-v3-origin-private/src/contracts/protocol/libraries/math/PercentageMath.sol
@@ -127,8 +127,24 @@
             )
         }
     }
 
+    function percentDivFloor(
+        uint256 value,
+        uint256 percentage
+    ) internal pure returns (uint256 result) {
+        // to avoid overflow, value <= type(uint256).max / PERCENTAGE_FACTOR
+        assembly {
+            if or(
+                iszero(percentage),
+                iszero(iszero(gt(value, div(not(0), PERCENTAGE_FACTOR))))
+            ) {
+                revert(0, 0)
+            }
+            result := div(mul(value, PERCENTAGE_FACTOR), percentage)
+        }
+    }
+
     function percentDivCeil(
         uint256 value,
         uint256 percentage
     ) internal pure returns (uint256 result) {
