===================================================================
--- lib/aave-v3-origin/src/contracts/instances/PoolInstance.sol
+++ lib/aave-v3-origin-private/src/contracts/instances/PoolInstance.sol
@@ -11,9 +11,9 @@
  * @author BGD Labs
  * @notice Instance of the Pool for the Aave protocol
  */
 contract PoolInstance is Pool {
-    uint256 public constant POOL_REVISION = 10;
+    uint256 public constant POOL_REVISION = 11;
 
     constructor(
         IPoolAddressesProvider provider,
         IReserveInterestRateStrategy interestRateStrategy_
