===================================================================
--- src/contracts/protocol/libraries/logic/FlashLoanLogic.sol
+++ lib/aave-v3-origin-private/src/contracts/protocol/libraries/logic/FlashLoanLogic.sol
@@ -154,12 +154,9 @@
                         oracle: IPoolAddressesProvider(params.addressesProvider)
                             .getPriceOracle(),
                         userEModeCategory: IPool(params.pool)
                             .getUserEMode(params.onBehalfOf)
-                            .toUint8(),
-                        priceOracleSentinel: IPoolAddressesProvider(
-                            params.addressesProvider
-                        ).getPriceOracleSentinel()
+                            .toUint8()
                     })
                 );
                 // no premium is paid when taking on the flashloan as debt
                 emit IPool.FlashLoan(
