===================================================================
--- src/contracts/protocol/libraries/configuration/UserConfiguration.sol
+++ lib/aave-v3-origin-private/src/contracts/protocol/libraries/configuration/UserConfiguration.sol
@@ -175,60 +175,8 @@
         return self.data == 0;
     }
 
     /**
-     * @notice Returns the Isolation Mode state of the user
-     * @param self The configuration object
-     * @param reservesData The state of all the reserves
-     * @param reservesList The addresses of all the active reserves
-     * @return True if the user is in isolation mode, false otherwise
-     * @return The address of the only asset used as collateral
-     * @return The debt ceiling of the reserve
-     */
-    function getIsolationModeState(
-        DataTypes.UserConfigurationMap memory self,
-        mapping(address => DataTypes.ReserveData) storage reservesData,
-        mapping(uint256 => address) storage reservesList
-    ) internal view returns (bool, address, uint256) {
-        if (isUsingAsCollateralOne(self)) {
-            uint256 assetId = _getFirstAssetIdByMask(self, COLLATERAL_MASK);
-
-            address assetAddress = reservesList[assetId];
-            uint256 ceiling = reservesData[assetAddress]
-                .configuration
-                .getDebtCeiling();
-            if (ceiling != 0) {
-                return (true, assetAddress, ceiling);
-            }
-        }
-        return (false, address(0), 0);
-    }
-
-    /**
-     * @notice Returns the siloed borrowing state for the user
-     * @param self The configuration object
-     * @param reservesData The data of all the reserves
-     * @param reservesList The reserve list
-     * @return True if the user has borrowed a siloed asset, false otherwise
-     * @return The address of the only borrowed asset
-     */
-    function getSiloedBorrowingState(
-        DataTypes.UserConfigurationMap memory self,
-        mapping(address => DataTypes.ReserveData) storage reservesData,
-        mapping(uint256 => address) storage reservesList
-    ) internal view returns (bool, address) {
-        if (isBorrowingOne(self)) {
-            uint256 assetId = _getFirstAssetIdByMask(self, BORROWING_MASK);
-            address assetAddress = reservesList[assetId];
-            if (reservesData[assetAddress].configuration.getSiloedBorrowing()) {
-                return (true, assetAddress);
-            }
-        }
-
-        return (false, address(0));
-    }
-
-    /**
      * @notice Returns the borrowed and collateral flags for the first asset on the bitmap and the bitmap shifted by two.
      * @dev This function mutates the input and the 2 bit slots in the bitmap will no longer correspond to the reserve index.
      * This is useful in situations where we want to iterate the bitmap as it allows for early exit once the bitmap turns zero.
      * @param data The configuration uint256
