Index: settings
===================================================================
--- settings
+++ settings
@@ -1,57 +1,58 @@
 {
+  "evmVersion": "cancun",
+  "libraries": {
+    "lib/aave-v3-origin-private/src/contracts/protocol/libraries/logic/BorrowLogic.sol": {
+      "BorrowLogic": "0x52Da0ce88202D1542543598D1e1e27F0d344726A"
+    },
+    "lib/aave-v3-origin-private/src/contracts/protocol/libraries/logic/FlashLoanLogic.sol": {
+      "FlashLoanLogic": "0x6D414cd0d5eAf8C43200ac0c325A7E2Ad83B8be6"
+    },
+    "lib/aave-v3-origin-private/src/contracts/protocol/libraries/logic/LiquidationLogic.sol": {
+      "LiquidationLogic": "0x96D5686812e33Ab509ECCDb38C89d15607B2a413"
+    },
+    "lib/aave-v3-origin-private/src/contracts/protocol/libraries/logic/PoolLogic.sol": {
+      "PoolLogic": "0xD70593841C57cbaA04957cc3EAcE95708e48853b"
+    },
+    "lib/aave-v3-origin-private/src/contracts/protocol/libraries/logic/SupplyLogic.sol": {
+      "SupplyLogic": "0x584C7d8c4cb05304FE5Ac7fbc97f20A10Fb07564"
+    }
+  },
+  "metadata": {
+    "appendCBOR": true,
+    "bytecodeHash": "none",
+    "useLiteralContent": false
+  },
+  "optimizer": {
+    "enabled": true,
+    "runs": 200
+  },
+  "outputSelection": {
+    "*": {
+      "": [
+        "*"
+      ],
+      "*": [
+        "*"
+      ]
+    }
+  },
   "remappings": [
     "aave-address-book/=lib/aave-helpers/lib/aave-address-book/src/",
     "aave-helpers/=lib/aave-helpers/",
-    "aave-v3-origin-tests/=lib/aave-v3-origin/tests/",
-    "aave-v3-origin/=lib/aave-v3-origin/src/",
+    "aave-v3-origin-tests/=lib/aave-v3-origin-private/tests/",
+    "aave-v3-origin/=lib/aave-v3-origin-private/src/",
     "erc4626-tests/=lib/aave-helpers/lib/solidity-utils/lib/openzeppelin-contracts-upgradeable/lib/erc4626-tests/",
-    "forge-std/=lib/forge-std/src/",
+    "forge-std/=lib/aave-helpers/lib/forge-std/src/",
     "openzeppelin-contracts-upgradeable/=lib/aave-helpers/lib/aave-address-book/lib/aave-v3-origin/lib/solidity-utils/lib/openzeppelin-contracts-upgradeable/",
     "openzeppelin-contracts/=lib/aave-helpers/lib/aave-address-book/lib/aave-v3-origin/lib/solidity-utils/lib/openzeppelin-contracts-upgradeable/lib/openzeppelin-contracts/",
     "solidity-utils/=lib/aave-helpers/lib/aave-address-book/lib/aave-v3-origin/lib/solidity-utils/src/",
     "lib/aave-helpers/:aave-address-book/=lib/aave-helpers/lib/aave-address-book/src/",
     "lib/aave-helpers/:solidity-utils/=lib/aave-helpers/lib/aave-address-book/lib/aave-v3-origin/lib/solidity-utils/src/",
-    "@openzeppelin/contracts-upgradeable/=lib/aave-v3-origin/lib/solidity-utils/lib/openzeppelin-contracts-upgradeable/contracts/",
-    "@openzeppelin/contracts/=lib/aave-v3-origin/lib/solidity-utils/lib/openzeppelin-contracts-upgradeable/lib/openzeppelin-contracts/contracts/",
-    "ds-test/=lib/aave-v3-origin/lib/forge-std/lib/ds-test/src/",
-    "halmos-cheatcodes/=lib/aave-v3-origin/lib/solidity-utils/lib/openzeppelin-contracts-upgradeable/lib/halmos-cheatcodes/src/"
+    "@openzeppelin/contracts-upgradeable/=lib/aave-v3-origin-private/lib/solidity-utils/lib/openzeppelin-contracts-upgradeable/contracts/",
+    "@openzeppelin/contracts/=lib/aave-v3-origin-private/lib/solidity-utils/lib/openzeppelin-contracts-upgradeable/lib/openzeppelin-contracts/contracts/",
+    "aave-v3-origin-private/=lib/aave-v3-origin-private/",
+    "ds-test/=lib/aave-v3-origin-private/lib/forge-std/lib/ds-test/src/",
+    "halmos-cheatcodes/=lib/aave-v3-origin-private/lib/solidity-utils/lib/openzeppelin-contracts-upgradeable/lib/halmos-cheatcodes/src/"
   ],
-  "optimizer": {
-    "enabled": true,
-    "runs": 200
-  },
-  "metadata": {
-    "useLiteralContent": false,
-    "bytecodeHash": "none",
-    "appendCBOR": true
-  },
-  "outputSelection": {
-    "lib/aave-v3-origin/src/contracts/instances/L2PoolInstance.sol": {
-      "L2PoolInstance": [
-        "*"
-      ]
-    }
-  },
-  "evmVersion": "cancun",
-  "viaIR": false,
-  "libraries": {
-    "lib/aave-v3-origin/src/contracts/protocol/libraries/logic/BorrowLogic.sol": {
-      "BorrowLogic": "0x1Fb8f7C906cddD28B42ae6Eb76abC04A91188635"
-    },
-    "lib/aave-v3-origin/src/contracts/protocol/libraries/logic/ConfiguratorLogic.sol": {
-      "ConfiguratorLogic": "0x6E2aFD57a161d12f34f416c29619BFeAcAC8AA18"
-    },
-    "lib/aave-v3-origin/src/contracts/protocol/libraries/logic/FlashLoanLogic.sol": {
-      "FlashLoanLogic": "0xE66AdcF153d0bD67a74e5d23e23F90b4011024EC"
-    },
-    "lib/aave-v3-origin/src/contracts/protocol/libraries/logic/LiquidationLogic.sol": {
-      "LiquidationLogic": "0x8dc095F287dBEBD9e15f75BB3DfBF18389BB8104"
-    },
-    "lib/aave-v3-origin/src/contracts/protocol/libraries/logic/PoolLogic.sol": {
-      "PoolLogic": "0x25Ed699a052B820d772765E26Ed4dD9f0A26F3E5"
-    },
-    "lib/aave-v3-origin/src/contracts/protocol/libraries/logic/SupplyLogic.sol": {
-      "SupplyLogic": "0x33654b16A4De97bCE05d7dD06803Bf1066f3123C"
-    }
-  }
+  "viaIR": false
 }
\ No newline at end of file
