===================================================================
--- src/contracts/protocol/libraries/logic/BorrowLogic.sol
+++ lib/aave-v3-origin-private/src/contracts/protocol/libraries/logic/BorrowLogic.sol
@@ -2,9 +2,9 @@
 pragma solidity ^0.8.10;
 
 import {GPv2SafeERC20} from "../../../dependencies/gnosis/contracts/GPv2SafeERC20.sol";
 import {SafeCast} from "openzeppelin-contracts/contracts/utils/math/SafeCast.sol";
-import {IERC20} from "../../../dependencies/openzeppelin/contracts/IERC20.sol";
+import {IERC20} from "openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
 import {IVariableDebtToken} from "../../../interfaces/IVariableDebtToken.sol";
 import {IAToken} from "../../../interfaces/IAToken.sol";
 import {IPool} from "../../../interfaces/IPool.sol";
 import {TokenMath} from "../../libraries/helpers/TokenMath.sol";
@@ -12,9 +12,8 @@
 import {ReserveConfiguration} from "../configuration/ReserveConfiguration.sol";
 import {DataTypes} from "../types/DataTypes.sol";
 import {ValidationLogic} from "./ValidationLogic.sol";
 import {ReserveLogic} from "./ReserveLogic.sol";
-import {IsolationModeLogic} from "./IsolationModeLogic.sol";
 
 /**
  * @title BorrowLogic library
  * @author Aave
@@ -30,10 +29,9 @@
     using SafeCast for uint256;
 
     /**
      * @notice Implements the borrow feature. Borrowing allows users that provided collateral to draw liquidity from the
-     * Aave protocol proportionally to their collateralization power. For isolated positions, it also increases the
-     * isolated debt.
+     * Aave protocol proportionally to their collateralization power.
      * @dev  Emits the `Borrow()` event
      * @param reservesData The state of all the reserves
      * @param reservesList The addresses of all the active reserves
      * @param eModeCategories The configuration of all the efficiency mode categories
@@ -57,20 +55,15 @@
         );
 
         ValidationLogic.validateBorrow(
             reservesData,
-            reservesList,
             eModeCategories,
             DataTypes.ValidateBorrowParams({
                 reserveCache: reserveCache,
-                userConfig: userConfig,
                 asset: params.asset,
-                userAddress: params.onBehalfOf,
                 amountScaled: amountScaled,
                 interestRateMode: params.interestRateMode,
-                oracle: params.oracle,
-                userEModeCategory: params.userEModeCategory,
-                priceOracleSentinel: params.priceOracleSentinel
+                userEModeCategory: params.userEModeCategory
             })
         );
 
         reserveCache.nextScaledVariableDebt = IVariableDebtToken(
@@ -87,16 +80,8 @@
         if (!userConfig.isBorrowing(cachedReserveId)) {
             userConfig.setBorrowing(cachedReserveId, true);
         }
 
-        IsolationModeLogic.increaseIsolatedDebtIfIsolated(
-            reservesData,
-            reservesList,
-            userConfig,
-            reserveCache,
-            params.amount
-        );
-
         reserve.updateInterestRatesAndVirtualBalance(
             reserveCache,
             params.asset,
             0,
@@ -133,10 +118,9 @@
     }
 
     /**
      * @notice Implements the repay feature. Repaying transfers the underlying back to the aToken and clears the
-     * equivalent amount of debt for the user by burning the corresponding debt token. For isolated positions, it also
-     * reduces the isolated debt.
+     * equivalent amount of debt for the user by burning the corresponding debt token.
      * @dev  Emits the `Repay()` event
      * @param reservesData The state of all the reserves
      * @param reservesList The addresses of all the active reserves
      * @param onBehalfOfConfig The user configuration mapping that tracks the supplied/borrowed assets
@@ -204,16 +188,8 @@
         if (noMoreDebt) {
             onBehalfOfConfig.setBorrowing(reserve.id, false);
         }
 
-        IsolationModeLogic.reduceIsolatedDebtIfIsolated(
-            reservesData,
-            reservesList,
-            onBehalfOfConfig,
-            reserveCache,
-            paybackAmount
-        );
-
         // in case of aToken repayment the sender must always repay on behalf of itself
         if (params.useATokens) {
             // As aToken.burn rounds up the burned shares, we ensure at least an equivalent of >= paybackAmount is burned.
             bool zeroBalanceAfterBurn = IAToken(reserveCache.aTokenAddress)
