===================================================================
--- src/contracts/interfaces/IERC20WithPermit.sol
+++ lib/aave-v3-origin-private/src/contracts/interfaces/IERC20WithPermit.sol
@@ -1,8 +1,8 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
-import {IERC20} from "../dependencies/openzeppelin/contracts/IERC20.sol";
+import {IERC20} from "openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
 
 /**
  * @title IERC20WithPermit
  * @author Aave
