===================================================================
--- src/contracts/interfaces/IInitializableDebtToken.sol
+++ lib/aave-v3-origin-private/src/contracts/interfaces/IInitializableDebtToken.sol
@@ -1,8 +1,7 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
-import {IAaveIncentivesController} from "./IAaveIncentivesController.sol";
 import {IPool} from "./IPool.sol";
 
 /**
  * @title IInitializableDebtToken
