===================================================================
--- src/contracts/protocol/libraries/logic/SupplyLogic.sol
+++ lib/aave-v3-origin-private/src/contracts/protocol/libraries/logic/SupplyLogic.sol
@@ -1,8 +1,8 @@
 // SPDX-License-Identifier: BUSL-1.1
 pragma solidity ^0.8.10;
 
-import {IERC20} from "../../../dependencies/openzeppelin/contracts/IERC20.sol";
+import {IERC20} from "openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
 import {GPv2SafeERC20} from "../../../dependencies/gnosis/contracts/GPv2SafeERC20.sol";
 import {IAToken} from "../../../interfaces/IAToken.sol";
 import {IPool} from "../../../interfaces/IPool.sol";
 import {Errors} from "../helpers/Errors.sol";
@@ -33,15 +33,13 @@
      * @dev Emits the `Supply()` event.
      * @dev In the first supply action, `ReserveUsedAsCollateralEnabled()` is emitted, if the asset can be enabled as
      * collateral.
      * @param reservesData The state of all the reserves
-     * @param reservesList The addresses of all the active reserves
      * @param userConfig The user configuration mapping that tracks the supplied/borrowed assets
      * @param params The additional parameters needed to execute the supply function
      */
     function executeSupply(
         mapping(address => DataTypes.ReserveData) storage reservesData,
-        mapping(uint256 => address) storage reservesList,
         mapping(uint8 => DataTypes.EModeCategory) storage eModeCategories,
         DataTypes.UserConfigurationMap storage userConfig,
         DataTypes.ExecuteSupplyParams memory params
     ) external {
@@ -83,14 +81,11 @@
         );
 
         if (isFirstSupply) {
             if (
-                ValidationLogic.validateAutomaticUseAsCollateral(
+                ValidationLogic.validateUseAsCollateral(
                     reservesData,
-                    reservesList,
                     eModeCategories,
-                    userConfig,
-                    reserveCache.reserveConfiguration,
                     params.asset,
                     params.supplierEModeCategory
                 )
             ) {
@@ -314,16 +309,13 @@
 
             require(
                 ValidationLogic.validateUseAsCollateral(
                     reservesData,
-                    reservesList,
                     eModeCategories,
-                    userConfig,
-                    reserveConfigCached,
                     asset,
                     userEModeCategory
                 ),
-                Errors.UserInIsolationModeOrLtvZero()
+                Errors.UserHasAssetWithZeroLtv()
             );
 
             userConfig.setUsingAsCollateral(reserve.id, asset, user, true);
         } else {
